src/sentry/notifications/platform/slack/provider.py

```diff
@@ -13,18 +24,37 @@
 )
 from sentry.organizations.services.organization.model import RpcOrganizationSummary
 
-# TODO(ecosystem): Figure out a way to use 'SlackBlock' type
-type SlackRenderable = Any
+
+class SlackRenderable(TypedDict):
+    blocks: list[Block]
 
 
 class SlackRenderer(NotificationRenderer[SlackRenderable]):
-    provider_key = NotificationProviderKey.DISCORD
+    provider_key = NotificationProviderKey.SLACK
 
     @classmethod
     def render[DataT: NotificationData](
         cls, *, data: DataT, rendered_template: NotificationRenderedTemplate
     ) -> SlackRenderable:
-        return {}
+        subject = HeaderBlock(text=PlainTextObject(text=rendered_template.subject))
+        body = SectionBlock(text=MarkdownTextObject(text=rendered_template.body))
+
+        actions_block = ActionsBlock(elements=[])
+        for action in rendered_template.actions:
+            actions_block.elements.append(
+                ButtonElement(text=action.get("label", ""), url=action.get("link", ""))
+            )
```

Comment:
We can just do `action["label"]` and `action["link"]` here, they're both required fields on the `NotificationRenderedAction` TypedDict.

I suppose you can also change `NotificationRenderedAction` if you just wanna access `.label` and `.link` directly. Might prevent the urge to do these `.get()` accesses altogether 👍